### PR TITLE
Work on login styling & accounts page

### PR DESCRIPTION
**Pull request by dm1031**
Ay ay ay boos German4 wut

### CODE REVIEW

File: src/client/src/App.tsx [hunk 1]
```diff
@@ -1,6 +1,8 @@
-import React from 'react'
+import React, { Props } from 'react'
```
Comment (hgranthorner) [inline src/client/src/App.tsx]: let's remove these weird things

File: src/client/src/components/Login/Login.tsx [hunk 2]
```diff
@@ -1,6 +1,6 @@
-import React, { useState } from 'react'
+import React, { useState, Dispatch } from 'react'
 import { login } from '../../store/thunks'
-import { connect } from 'react-redux'
+import { connect, DispatchProp } from 'react-redux'
```
Comment (hgranthorner) [inline src/client/src/components/Login/Login.tsx]: what is this

File: src/client/src/App.tsx (hunk 1 above)
Comment (dm1031) [reply src/client/src/App.tsx]: ay ay deleted

File: src/client/src/components/Login/Login.tsx (hunk 2 above)
Comment (dm1031) [reply src/client/src/components/Login/Login.tsx]: ay ay deleted

**Review by hgranthorner — APPROVED** [review]
me gusta

**Pull request merged**